Altis_Life.Altis/core/actions/fn_storeVehicle.sqf

```diff
@@ -8,9 +8,9 @@
 */
 private ["_nearVehicles","_vehicle"];
 if !(isNull objectParent player) then {
-    _vehicle = vehicle player;
+    private _vehicle = vehicle player;
 } else {
-    _nearVehicles = nearestObjects[getPos (_this select 0),["Car","Air","Ship"],30]; //Fetch vehicles within 30m.
+   private  _nearVehicles = nearestObjects[getPos (_this select 0),["Car","Air","Ship"],30]; //Fetch vehicles within 30m.
```

Comment:
Might be worth changing the ``_this select 0``, to param (for readability with types) or stick in a params at top